src/session/parse_enum.rs

```diff
@@ -0,0 +1,61 @@
+use crate::session::UnknownEnumValueError;
+
+pub trait ProtoEnum<Proto> {
+    /// Returns a default value for given Protobuf definition.
+    fn default() -> Proto;
```

Comment:
This is covered by the standard `Default` trait.

Reply:
I think my previous answer covers this comment also.

Reply:
I don't think it does (if it does, then please, explain).

Reply:
The default value of any proto enum must be 0 ([link](https://developers.google.com/protocol-buffers/docs/proto3#:~:text=For%20enums%2C%20the%20default%20value,exact%20value%20is%20language%2Ddependent.)) In our case we care about the value to which we default when given proto enum is not set.